src/main/java/appeng/client/gui/AEBaseScreen.java

```diff
@@ -656,7 +638,7 @@ protected void moveItems(MatrixStack matrices, Slot s) {
             try {
                 renderAppEngSlot(matrices, (AppEngSlot) s);
             } catch (final Exception err) {
-                AELog.warn("[AppEng] AE prevented crash while drawing slot: " + err.toString());
+                AELog.warn("[AppEng] AE prevented crash while drawing slot: " + err);
```

Comment:
A bit OT, but we should check at some point about removing `[AppEng]` or similar tags. The log should already contain the actual logger name. I'd consider doing it afterwards, to avoid merge conflicts now.

Reply:
Yeah, I have no idea where that is coming from.